metadata-ingestion/tests/integration/ldap/ldap_mce_golden.json

```diff
@@ -1,106 +1,125 @@
 [
-{
+  {
     "auditHeader": null,
     "proposedSnapshot": {
-        "com.linkedin.pegasus2avro.metadata.snapshot.CorpUserSnapshot": {
-            "urn": "urn:li:corpuser:bsimpson",
-            "aspects": [
-                {
-                    "com.linkedin.pegasus2avro.identity.CorpUserInfo": {
-                        "active": true,
-                        "displayName": "Bart Simpson",
-                        "email": "bsimpson",
-                        "title": "Mr. Boss",
-                        "managerUrn": null,
-                        "departmentId": null,
-                        "departmentName": null,
-                        "firstName": "Bart",
-                        "lastName": "Simpson",
-                        "fullName": "Bart Simpson",
-                        "countryCode": null
-                    }
-                }
-            ]
-        }
+      "com.linkedin.pegasus2avro.metadata.snapshot.CorpGroupSnapshot": {
```

Comment:
This is actually the only one added.

Reply:
Can add more tests with additional admins - members if needed.

Reply:
I'm pretty happy with the current level of testing: unit tests + integration + working internally at your company - seems pretty comprehensive 😄 

For future reference, there's a `scripts/update_golden_files.sh` that you can use to regenerate these golden MCE files, and then just check that the resulting diff looks reasonable